packages/ra-ui-materialui/src/input/SelectArrayInput.stories.tsx

```diff
@@ -1,28 +1,31 @@
-import * as React from 'react';
-import polyglotI18nProvider from 'ra-i18n-polyglot';
-import englishMessages from 'ra-language-english';
+import CloseIcon from '@mui/icons-material/Close';
 import {
+    Box,
     Button,
     Dialog,
     DialogActions,
     DialogContent,
+    DialogTitle,
+    IconButton,
     Stack,
-    Box,
     TextField,
 } from '@mui/material';
 import fakeRestProvider from 'ra-data-fakerest';
+import polyglotI18nProvider from 'ra-i18n-polyglot';
+import englishMessages from 'ra-language-english';
+import * as React from 'react';
 
+import { CreateBase, Resource, TestMemoryRouter } from 'ra-core';
+import { Admin } from 'react-admin';
```

Comment:
Please do not import from `'react-admin'` as it will create a circular dependency